pkg/util/math.go

```diff
@@ -19,3 +20,12 @@ func Max(x, y int64) int64 {
 func MaxDuration(x, y time.Duration) time.Duration {
 	return time.Duration(Max(int64(x), int64(y)))
 }
+
+func GenerateAlphaNumeric(len int) string {
```

Comment:
That already exists in `util.GeneratePassword()`.

Reply:
Done